src/main/java/com/ericsson/ei/controller/RuleController.java

```diff
@@ -45,7 +52,7 @@
      * This method checks if the possibility to test rules has been enabled in Eiffel Intelligence.
      * 
      */
-    @RequestMapping(value = "/testRulePageEnabled", method = RequestMethod.GET)
+    @RequestMapping(value = "/rule-check/testRulePageEnabled", method = RequestMethod.GET)
```

Comment:
Could not "enabled" be enough for the last part of the endpoint? It sounds less repetitive.

Reply:
I don't want to touch naming and path structure since people had strong opinions on this.

Reply:
Fair enough, maybe something to keep in mind for later then